patches/tModLoader/Terraria/DataStructures/PlayerDrawLayers.cs.patch

```diff
@@ -293,9 +315,12 @@
  			if (drawinfo.drawPlayer.babyBird) {
  				Rectangle bodyFrame5 = drawinfo.drawPlayer.bodyFrame;
  				bodyFrame5.Y = 0;
-@@ -1856,7 +_,7 @@
+@@ -1854,9 +_,9 @@
+ 		}
+ 
  		private static void DrawPlayer_21_Head_TheFace(ref PlayerDrawSet drawinfo) {
- 			bool flag = drawinfo.drawPlayer.head == 38 || drawinfo.drawPlayer.head == 135 || drawinfo.drawPlayer.head == 269;
+-			bool flag = drawinfo.drawPlayer.head == 38 || drawinfo.drawPlayer.head == 135 || drawinfo.drawPlayer.head == 269;
++			bool flag = drawinfo.drawPlayer.head == 38 || drawinfo.drawPlayer.head == 135 || drawinfo.drawPlayer.head == 269 || !ItemLoader.DrawHead(drawinfo.drawPlayer);
  			DrawData item;
```

Comment:
A lot of these draw hooks should be removed in favor of ItemID.Sets.* entries, and additional booleans for PlayerDrawSet. There should be some custom drawing-related sets entries already, they can serve as an example. The original vanilla ids are added to the sets, the line checking is commented out, and an edited line that just uses the new boolean from PlayerDrawSets is added. The booleans are reset somewhere around BoringSetup based on the values from sets. The sets entries also should have a note for where in the source code do the vanilla ids come from.